src/styles/components/extra-tips.scss

```diff
@@ -0,0 +1,69 @@
+.extra-tips {
+  justify-content: center;
+  margin-top: 30px;
+
+  &__title {
+    padding-bottom: 18px;
+    margin-bottom: 22px;
+    font-weight: 900;
+    line-height: 42px;
+    color: $semiblack;
+    text-align: center;
+    border-bottom: 1px solid $silver;
+  }
+}
+
+.extra-tip {
+  justify-content: center;
+  margin: 15px 0;
+
+  &__element-aside {
+    display: inline-block;
+
+    @include media-breakpoint-up(lg) {
+      float: left;
+    }
+  }
+
+  &__element-header {
+    display: inline-block;
+
+    @include media-breakpoint-up(lg) {
+      float: left;
+    }
+  }
+
+  &__element-content {
+    @include media-breakpoint-up(lg) {
+      margin-top: 40px;
+    }
+  }
+
+  &__icon {
+    @include size(45px,45px);
```

Comment:
why 2 arguments?

Reply:
Fixed (forgot that $height has $width as default value )